roles/kubernetes/secrets/tasks/sync_kube_master_certs.yml

```diff
@@ -32,7 +32,7 @@
     sync_file_hosts: "{{ groups['kube-master'] }}"
     sync_file_is_cert: true
     sync_file_owner: kube
-  with_items: ["apiserver.pem", "kube-scheduler.pem", "kube-controller-manager.pem", "front-proxy-client.pem"]
+  with_items: ["apiserver.pem", "kube-scheduler.pem", "kube-controller-manager.pem"]
```

Comment:
it really should be in this category, but our logic is bad for generating the missing file if you already have apiserver
cert.